rviz_common/CMakeLists.txt

```diff
@@ -32,6 +32,10 @@ if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
   endif()
 endif()
 
+if(${CMAKE_SYSTEM_NAME} MATCHES "Linux" AND ${CMAKE_CXX_COMPILER_ID} MATCHES "Clang")
+    link_libraries(stdc++fs)
```

Comment:
Which library needs `stdc++fs`? Does `target_link_libraries(rviz_common stdc++fs)` resolve the issue?

Reply:
Yeah, pleasae narrow this down to the specific library that needs it.

Reply:
It may be the case as William and Tully pointed out that pluginlib is the one which is dependent on std filesystem. Will check if I can update my other PR for pluginlib to handle the library requirement here: [ros/pluginlib#144](https://github.com/ros/pluginlib/pull/144)

Reply:
Adding the library as an export in pluginlib worked like a charm! Thanks a lot William and Tully! Please follow [ros/pluginlib#144 ](https://github.com/ros/pluginlib/pull/144) if needed as I will discard this PR now